rules/windows/credential_access_veeam_commands.toml

```diff
@@ -0,0 +1,74 @@
+[metadata]
+creation_date = "2024/03/14"
+integration = ["windows", "endpoint", "system"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2024/03/14"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies commands that can access and decrypt Veeam credentials stored in MSSQL databases. Attackers can use Veeam
+Credentials to target backups as part of destructive operations such as Ransomware attacks.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.process-*", "logs-windows.*", "endgame-*", "logs-system.security*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential Veeam Credential Access Command"
+references = [
+  "https://thedfirreport.com/2021/12/13/diavol-ransomware/"
+]
+risk_score = 47
+rule_id = "b661f86d-1c23-4ce7-a59e-2edbdba28247"
+severity = "medium"
+tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Defense Evasion", "Tactic: Credential Access", "Data Source: Elastic Endgame", "Data Source: Elastic Defend"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where host.os.type == "windows" and event.type == "start" and
+  (
+    (process.name : "sqlcmd.exe" or process.pe.original_file_name : "sqlcmd.exe") or
+    process.args : ("Invoke-Sqlcmd", "Invoke-SqlExecute", "Invoke-DbaQuery", "Invoke-SqlQuery")
+  ) and
+  process.args : "*[VeeamBackup].[dbo].[Credentials]*"
```

Comment:
```suggestion
  stringContains(process.args, "[VeeamBackup].[dbo].[Credentials]")
```

Reply:
Leaving this one out for now, until we do a ruleset-wide review after we can measure how effective this is